main/cicd/cicd-pipeline/config/infra/cloudformation.yml

```diff
@@ -346,6 +343,11 @@ Resources:
           - Action:
               - iam:PutRolePermissionsBoundary
               - iam:DeleteRolePermissionsBoundary
+              - iam:CreateRole
+              - iam:PutRolePolicy
+              - iam:AttachRolePolicy
+              - iam:DeleteRolePolicy
+              - iam:DetachRolePolicy
             Effect: Deny
             Resource: !Sub 'arn:aws:iam::${AWS::AccountId}:role/${self:custom.settings.namespace}-cicd-pipeline-AppDeployerRole-*'
```

Comment:
Should there be two arns- role and policy?

Reply:
No, but doesn't hurt to explicitly deny. Added.